main/ui/src/main/java/org/cryptomator/ui/util/mount/LinuxGvfsWebDavMounter.java

```diff
@@ -70,4 +74,11 @@ public void unmount() throws CommandFailedException {
 		};
 	}
 
+	private Script openFMWithWebdavSchema(String schemaName, URI uri){
+		return Script.fromLines(
+				"set -x",
+				"xdg-open \""+schemaName+":$DAV_SSP\"")
```

Comment:
maybe you can use `$DAV_SCHEME` together with `.addEnv("DAV_SCHEME", schemaName)`.

Or even build a new URI with the scheme already in it and pass a single arg to this function?